### ISSUE TITLE
Inconsistancy when re-running the code on A100

### ISSUE THREAD

**Issue by LindaLydia** [FuChong-cyber/label-inference-attacks#6]
In your paper, you reported that for CIFAR-10 dataset, the main task accuracy is 0.8280 while the passieve model completion attack performance is 0.8024 on the training dataset and 0.6299 on the testing dataset with 40 auxiliary labeled data in total. 

However, when re-running your code on my device (A100) without any modification of the default hyper-parameter settings, I got 0.8015 for main task performance, which is similar to your reported result, but 0.8721 on training dataset and 0.6984 on testing dataset for attack accuracy, which is apparently too high compared to your reported results. 

Could you help me explain why this could happen?

**Comment by mehwish-aman** [comment]
Which system did you use for experimenting this code?

**Comment by FuChong-cyber** [comment]
The clean and trojaned models are trained on four NVIDIA RTX 2080Ti GPU cards, Ubuntu system. 

Experiments about trojan detection are accelerated by one NVIDIA RTX 3090 GPU card, Windows system. 

All DNN models are implemented using PyTorch.



-----原始邮件-----
发件人:mehwish-aman ***@***.***>
发送时间:2023-07-11 15:11:04 (星期二)
收件人: FuChong-cyber/label-inference-attacks ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [FuChong-cyber/label-inference-attacks] Inconsistancy when re-running the code on A100 (Issue #6)






Which system did you use for experimenting this code?

**Comment by mehwish-aman** [comment]
> The clean and trojaned models are trained on four NVIDIA RTX 2080Ti GPU cards, Ubuntu system. Experiments about trojan detection are accelerated by one NVIDIA RTX 3090 GPU card, Windows system. All DNN models are implemented using PyTorch.
> […](#)
> -----原始邮件----- 发件人:mehwish-aman ***@***.***> 发送时间:2023-07-11 15:11:04 (星期二) 收件人: FuChong-cyber/label-inference-attacks ***@***.***> 抄送: Subscribed ***@***.***> 主题: Re: [FuChong-cyber/label-inference-attacks] Inconsistancy when re-running the code on A100 (Issue #6) Which system did you use for experimenting this code? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Thankyou so much for replying. I will further asked some questions if needed. Thanks again.

**Comment by LindaLydia** [comment]
Sorry to miss up the information on the hardware. I trained the model and did the completion attack both on A100.

**Comment by LindaLydia** [comment]
Maybe this difference is caused by the hardware difference... But is this amount of difference acceptable?
